hack/builder/publish.sh

```diff
@@ -7,5 +8,12 @@ SCRIPT_DIR="$(
 
 . ${SCRIPT_DIR}/version.sh
 
-docker tag kubevirt/builder:${VERSION} docker.io/kubevirt/builder:${VERSION}
-docker push docker.io/kubevirt/builder:${VERSION}
+for ARCH in ${ARCHITECTURES}; do
+    docker push kubevirt/builder:${VERSION}-${ARCH}
+    TMP_IMAGES="${TMP_IMAGES} kubevirt/builder:${VERSION}-${ARCH}"
+done
+
+export DOCKER_CLI_EXPERIMENTAL=enabled
+docker manifest create --amend kubevirt/builder:${VERSION} ${TMP_IMAGES}
+docker manifest push kubevirt/builder:${VERSION}
+rm manifests/ -rf
```

Comment:
This will not be called if we we create the manifest and fail to push it

Reply:
I can move the cleanup into a trap ...